Bottom: 41b78fe01f7cb66641bb006dd5b724d2474f5764
Top:    a8798ad670b827ef1459aa30759196a2aa7bce48
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-01 22:20:06 +0100

Change scroll-wheel bindings to match more GTK applications









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index f9e425a..2aab550 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -25,18 +25,18 @@ Mouse =
     up-shift-ctrl = { Mode(Notify) Mode(Restore) Display(ToggleRubberbandMode) }
   }
   Up = {
-    Zoom(0.8)
-    shift = Scroll(up)
-    ctrl = Scroll(left)
+    Scroll(up)
+    shift = Scroll(left)
+    ctrl = Zoom(0.8)
   }
   Down = {
-    Zoom(1.25)
-    shift = Scroll(down)
-    ctrl = Scroll(right)
+    Scroll(down)
+    shift = Scroll(right)
+    ctrl = Zoom(1.25)
   }
 # If you want zoom to center, do this instead.
-  #Up = { {Zoom(0.8) Center()} }
-  #Down = { {Zoom(1.25) Center()} }
+  #Up = { ctrl = {Zoom(0.8) Center()} }
+  #Down = { ctrl = {Zoom(1.25) Center()} }
 }
 
 MainMenu =
